client/src/components/generic/Scorecard/index.tsx

```diff
@@ -123,7 +123,8 @@ export default class Scorecard extends GenericComponent<IScorecardProps, any> {
       return;
     }
 
-    event.preventDefault();
+    let evt = typeof event !== 'undefined' && event || arguments[2];
```

Comment:
write event instead evt (:

Reply:
This will use a system saved variable which is not recommended. I used 'eventObject' instead